test/e2e/cypress/e2e/sap_systems_overview.cy.js

```diff
@@ -291,4 +291,22 @@ context('SAP Systems Overview', () => {
       cy.get('table.table-fixed').should('not.contain', 'DAA');
     });
   });
+
+  describe('Deregistration', () => {
+    const sap_system_nwp = {
+      sid: 'NWP',
+      hana_primary: {
+        name: 'vmhdbprd01',
+        id: '9cd46919-5f19-59aa-993e-cf3736c71053',
+      },
+    };
+
```

Comment:
I miss a test here where we check that the SAP system is deregistered when the ASCS or PAS instances are deregistered